lib/create-jws.js

```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright (c) 2020-2021 Digital Bazaar, Inc. All rights reserved.
+ */
+import * as env from './env.js';
+import * as forge from 'node-forge';
+
+export function createJwsFactory() {
+  if(env.nodejs) {
```

Comment:
This isn't a pattern we use / is an old pattern. The right way to do this is to have separate files for the browser version and for the node.js version -- and use an browser alias instead of putting things in one file and setting polyfills to `false`.

Reply:
Agreed, yeah. It's only there because it was extracted from the previous version of jsigs

Reply:
Removed.